ERCS/eip-####.md

```diff
@@ -0,0 +1,265 @@
+---
+eip: ####
```

Comment:
```suggestion
eip: 7772
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.